test/helpers.bash

```diff
@@ -434,8 +439,7 @@ function cleanup_test() {
 	cleanup_pods
 	stop_crio
 	cleanup_lvm
-	# Best effort remove the test dir, as sometime it is still busy
-	rm -rf "$TESTDIR" || true
+	rm -r "$TESTDIR" || true
```

Comment:
Is this still an issue, so can we remove the `|| true` now?

Reply:
unfortunately it is still an issue

Reply:
Okay, I will link it in https://github.com/cri-o/cri-o/issues/2665 Let's get these cleanups in nevertheless :)